src/main/java/edu/greenblitz/tobyDetermined/commands/MultiSystem/FullGrip.java

```diff
@@ -0,0 +1,31 @@
+package edu.greenblitz.tobyDetermined.commands.MultiSystem;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.commands.intake.extender.ExtendRoller;
+import edu.greenblitz.tobyDetermined.commands.intake.extender.RetractRoller;
+import edu.greenblitz.tobyDetermined.commands.intake.roller.RunRoller;
+import edu.greenblitz.tobyDetermined.commands.rotatingBelly.RotateOutDoorDirection;
+import edu.greenblitz.tobyDetermined.commands.telescopicArm.GoToPosition;
+import edu.greenblitz.tobyDetermined.subsystems.RotatingBelly.RotatingBelly;
+import edu.greenblitz.tobyDetermined.subsystems.intake.IntakeRoller;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class FullGrip extends SequentialCommandGroup {
+
+    public FullGrip() {
+        addCommands(
+                new ParallelCommandGroup(
+                        new ExtendRoller().alongWith(new RunRoller())
+                                .until(() -> RotatingBelly.getInstance().isObjectIn() || IntakeRoller.getInstance().isObjectIn()),
+                        new RotateOutDoorDirection())
+        );
+
+        addCommands(
```

Comment:
Raz - just a note in general (you don't have to fix it), the purpose of addCommands is to get as a parameter more than one command, then you don't have to use addCommands more than once per command group

Reply:
ok, noted